include/quic/utils.hpp

```diff
@@ -147,7 +159,12 @@ namespace oxen::quic
     // Error code we send to a stream close callback if the stream's connection expires
     inline constexpr uint64_t STREAM_ERROR_CONNECTION_EXPIRED = (1ULL << 62) + 1;
 
-    // bstring_view literals baby
+    // strang literals
+    inline std::basic_string_view<unsigned char> operator""_usv(const char* __str, size_t __len) noexcept
+    {
+        return std::basic_string_view<unsigned char>(reinterpret_cast<const unsigned char*>(__str), __len);
```

Comment:
This could use the `ustring_view` typedef for convenience.  (And a `bstring_view` for the one below).

Reply:
Nice they were even at the top, changed